cfme/fixtures/pytest_selenium.py

```diff
@@ -814,12 +814,25 @@ def force_navigate(page_name, _tries=0, *args, **kwargs):
         kwargs.pop("start", None)
         force_navigate("dashboard")  # Start fresh
 
-    try:
-        # What we'd like to happen...
+    def _login_func():
         if not current_user:  # default to admin user
             login.login_admin()
         else:  # we recycled and want to log back in
             login.login(current_user.username, current_user.password)
+
+    try:
+        try:
+            # What we'd like to happen...
+            _login_func()
+        except WebDriverException as e:
+            if "jquery" not in str(e).lower():
+                raise  # Something unknown happened
+            logger.info("Seems we got a non-CFME page (blank or screwed up) so killing the browser")
+            browser().quit()
+            # And try it again
+            _login_func()
+            # If this failed, no help with that :/
+
```

Comment:
This can't be in the next block where other errors are caught, because the jquery error comes right from these login calls.